integration-test/mqtt/mqtt_system_test.c

```diff
@@ -1123,9 +1120,6 @@ void test_MQTT_Restore_Session_Resend_PubRel( void )
                        MQTT_ProcessLoop( &context, 2 * MQTT_PROCESS_LOOP_TIMEOUT_MS ) );
     TEST_ASSERT_FALSE( receivedPubComp );
 
-    /* Verify that the connection with the broker has been disconnected. */
-    TEST_ASSERT_EQUAL( MQTTNotConnected, context.connectStatus );
-
```

Comment:
In the other tests, we reset `disconnectOnPacketType` back to `MQTT_PACKET_ID_INVALID` after resuming the session. I think we should do the same here, even if it doesn't affect the test.

Reply:
Good suggestion. Done.